src/Congo.WebApi/Data/ProductAccess/InsertProductHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Congo.WebApi.Data.Models;
+using MediatR;
+
+namespace Congo.WebApi.Data.ProductAccess
+{
+    public class InsertProductHelper : IRequestHandler<InsertProductCommand, Product>
```

Comment:
This is a IRequestHandler - someone that handles a request, probably wanna change Helper to Handler